scripts/regenerate-riot-latest.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -e
+
+DDTEST_CMD=scripts/ddtest
+
+pkgs=$(python scripts/freshvenvs.py | cut -d':' -f1)
+echo $pkgs
+
+if ! $DDTEST_CMD; then
+    echo "Command '$DDTEST_CMD' failed."
+    exit 1
+fi
+
+for pkg in ${pkgs[*]}; do
```

Comment:
What if we limit this to just a few packages per PR and then have another manual step that can do the entire list of packages?